locale/en/index.md

```diff
@@ -1,20 +1,3 @@
 ---
```

Comment:
Should we perhaps keep a backup of this so it is easier to change later?

Reply:
Shouldn't we just do a bit revert? It's git anyway.

Reply:
we can do that... but may want to keep the Black-lives-matter page as historical context. Easy enough to deal with later